realm/src/main/java/io/realm/dynamic/DynamicRealmObject.java

```diff
@@ -462,6 +462,11 @@ public int hashCode() {
     }
 
     @Override
+    protected void updateColumnInfo(ColumnInfo info) {
```

Comment:
No need to override this. This method should never be called on DynamicRealmObject?